contrib/testgen/gen_base58_test_vectors.py

```diff
@@ -27,14 +27,14 @@
 templates = [
   # prefix, payload_size, suffix, metadata
   #                                  None = N/A
-  ((PUBKEY_ADDRESS,),      20, (),   (False, False, 'pubkey', None)),
-  ((SCRIPT_ADDRESS,),      20, (),   (False, False, 'script',  None)),
-  ((PUBKEY_ADDRESS_TEST,), 20, (),   (False, True,  'pubkey', None)),
-  ((SCRIPT_ADDRESS_TEST,), 20, (),   (False, True,  'script',  None)),
-  ((PRIVKEY,),             32, (),   (True,  False, None,  False)),
-  ((PRIVKEY,),             32, (1,), (True,  False, None,  True)),
-  ((PRIVKEY_TEST,),        32, (),   (True,  True,  None,  False)),
-  ((PRIVKEY_TEST,),        32, (1,), (True,  True,  None,  True))
+  (PUBKEY_ADDRESS,        20, (),   (False, False, 'pubkey', None)),
+  (SCRIPT_ADDRESS,        20, (),   (False, False, 'script',  None)),
+  (PUBKEY_ADDRESS_TEST,   20, (),   (False, True,  'pubkey', None)),
+  (SCRIPT_ADDRESS_TEST,   20, (),   (False, True,  'script',  None)),
+  ((PRIVKEY,),            32, (),   (True,  False, None,  False)),
+  ((PRIVKEY,),            32, (1,), (True,  False, None,  True)),
+  ((PRIVKEY_TEST,),       32, (),   (True,  True,  None,  False)),
+  ((PRIVKEY_TEST,),       32, (1,), (True,  True,  None,  True))
```

Comment:
Darn, I was off-by-one in my suggested changes, and these four lines still have a trivial change. If you do squash the commits in this PR together, please also add an extra space to all eight lines so that only the first four show up in the diff.